code/modules/mob/living/brain/brain_item.dm

```diff
@@ -221,6 +244,19 @@
 			else
 				return brain_message
 
+/obj/item/organ/brain/before_organ_replacement(obj/item/organ/replacement)
+	. = ..()
+	var/obj/item/organ/brain/replacement_brain = replacement
+	if(!istype(replacement_brain))
+		return
+	for(var/obj/item/skillchip/S in src)
+		if(S in skillchips)
+			if(owner)
+				S.on_removal(owner)
+			LAZYREMOVE(skillchips,S)
+			LAZYADD(replacement_brain.skillchips,S) //No need to call on_apply here, since it will be inserted in organ replacement soon.
+		S.forceMove(replacement)
```

Comment:
```suggestion
	for(var/obj/item/skillchip/skill_chip in src)
		if(skill_chip in skillchips)
			if(owner)
				skill_chip.on_removal(owner)
			LAZYREMOVE(skillchips, skill_chip)
			LAZYADD(replacement_brain.skillchips, skill_chip) //No need to call on_apply here, since it will be inserted in organ replacement soon.
		skill_chip.forceMove(replacement)
```